mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1035,6 +1036,50 @@ func TestClientSideEncryptionProse(t *testing.T) {
 			})
 		}
 	})
+
+	// These tests only run when a KMS mock server is running on localhost:8000.
```

Comment:
Should we consider passing in the KMS server address as in an environment variable to make this more flexible? It'd also be consistent with how we pass in the cluster URI as `MONGODB_URI`.

Reply:
Since we're only using one KMS server address right now, a `KMS_URI` env variable sounds a little confusing. I think we can definitely add this if more KMS mock server instances are used in the future. Thoughts?

Reply:
I don't have a strong preference here. My thought was that someone might have a server running locally on port 8000 and that would block them from running this test, but I don't really expect that to happen. It's probably fine as-is and we can revisit if we run into issues.